src/metabase/server/middleware/exceptions.clj

```diff
@@ -45,11 +45,12 @@
 (defmethod api-exception-response Throwable
   [^Throwable e]
   (let [{:keys [status-code], :as info} (ex-data e)
-        other-info                      (dissoc info :status-code :schema :type)
+        other-info                      (dissoc info :status-code :schema :type :toucan2/context-trace)
         body                            (cond
-                                          (and status-code (empty? other-info))
-                                          ;; If status code was specified but other data wasn't, it's something like a
-                                          ;; 404. Return message as the (plain-text) body.
+                                          (and status-code (not= status-code 500) (empty? other-info))
```

Comment:
A few of those `check-500` endpoints were almost impossible to debug because it omitted the stacktrace. For 500s, since they're presumably unexpected errors, the stacktrace is important. So I tweaked this to include the stacktrace when we get a 500.